packages/app-config-writer/test/unit/variants-config/generateVariantsConfig.test.ts

```diff
@@ -18,7 +18,7 @@ describe('generateVariantsConfig', () => {
     test('adding variants configuration to a non existing project', async () => {
         const basePath = join(__dirname, '../../fixtures/a-folder-that-does-not-exist');
         await expect(generateVariantsConfig(basePath, 'hugo.yaml', new ToolsLogger())).rejects.toThrowError(
-            `Script 'start-variants-management' cannot be written to package.json. File 'package.json' not found at ${basePath}`
+            `File 'hugo.yaml' not found in project '${basePath}'`
```

Comment:
nit pick: why is the "Script 'start-variants-management' cannot be written to package.json." part gone?

Reply:
I [switched the methods](https://github.com/SAP/open-ux-tools/pull/2525/files#diff-9beb13032ed12a024e5c1dcfa70bdbbc0199fc10307b86633630b110c3a22227R27) to first update the yaml file accordingly. Therefore, the [error for a non-existing file](https://github.com/SAP/open-ux-tools/pull/2525/files#diff-40eac4eaa926a6b933a36d284af6c388f8f2668744420dca457fa61451f605f5R67) occurs at an earlier stage and not while handling the script insertion anymore.